### PR TITLE
Update US.js

### PR DESCRIPTION
**Pull request by justins83**
Changing problem solution description to match USA roundabout naming guidance

### CODE REVIEW

File: US.js [hunk 1]
```diff
@@ -21,3 +21,10 @@
     "problemEN": "The street name has incorrect abbreviation, or character",
     "solutionEN": "Check upper/lower case, a space before/after the abbreviation and the accordance with the abbreviation table. Remove any comma (,), backslash (\\), or semicolon (;)"
 },
+"29.enabled": true,
+"29.severity": 'W',
+"29.title": "Street name on roundabout",
+"29.problem": "Verify if roundabout should be named",
+"29.problemLink": "P:USA/Roundabout#Creation_from_an_intersection",
+"29.solution": "If the roundabout doesn't have a name, which is usually the case, click the None box next to Street. If the roundabout is a named circle on local signs, its segments can be named just like any other road.",
+"29.solutionLink": "P:Global/Creating_and_editing_road_segments#Address_Properties",
```
Comment (berestovskyy) [inline US.js:30]: Maybe, it's better to rephrase for all the countries, i.e. change the `default.js` instead? WDYT?

For the US-specific links, there is no need to duplicate the `29.enabled`, `29.title` etc. Whatever is not lister in `US.js` will be automatically taken from the `default.js`.

Comment (justins83) [reply US.js:30]: I cannot comment/make changes for all countries as I do not know what processes they follow for roundabouts, which is why I am only modifying for the US.

Thank you for finding those duplicates, I worked on this for a few minutes at a time throughout my work day and forgot to delete the portions that weren't necessary to repeat :)

**Review by dfinity-berestovskyy — APPROVED** [review]

File: US.js (hunk 1 above)
Comment (dfinity-berestovskyy) [reply US.js:27]: Thanks for you contribution, Justin!

**Review by dfinity-berestovskyy — APPROVED** [review]

File: US.js (hunk 1 above)
Comment (berestovskyy) [reply US.js:27]: Thanks for your contribution, Justin!

**Review by berestovskyy — APPROVED** [review]

**Pull request merged**